pyproject.toml

```
@@ -32,7 +32,7 @@ classifiers = [
 ]
 
 [project.urls]
-Homepage = "https://bitbucket.csiro.au/projects/SC/repos/sensor-api-python-client/browse"
+Homepage = "https://github.com/eratosio/sensor-api-python-client/tree/eratos-master"
```

```suggestion
Homepage = "https://github.com/eratosio/sensor-api-python-client"
```
Should always point to the default branch.